webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/AxisOrder.scala

```diff
@@ -97,6 +97,12 @@ case class FullAxisOrder(axes: Seq[Axis]) {
   def permuteIndicesArrayToWk(indices: Array[Int]): Array[Int] =
     arrayToWkPermutation.map(indices(_))
 
+  def toWkLibsDictObject: JsValue =
```

Comment:
```suggestion
  def toWkLibsDictJson: JsValue =
```

Reply:
I also went for `toWkLibsJson` here as below to have the naming more consistent. Feel free to argue